src/views/Farms/Farms.tsx

```diff
@@ -312,7 +312,7 @@ const Farms: React.FC = ({ children }) => {
   })
 
   const renderContent = (): JSX.Element => {
-    if (viewMode === ViewMode.TABLE && rowData.length) {
```

Comment:
we should change the test to make the code passed instead of changing the code to make it passed.

Reply:
Done